### PR TITLE
Rowe-1954-232

### PR DESCRIPTION
**Pull request by CarolinHu**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

#892. @LinguList, you are already involved and probably want to have a look at the conceptlist descriptions for this list, Morse and Mountain (to be added), would you mind being the moderator again?

### CODE REVIEW

**Review by LinguList — DISMISSED** [review]
@CarolinHu, the last sentence in the NOTE is no longer required: we have the PDF, did I share it? If not, can you remind me via email? I'll also prepare a PDF for the Concepticon. This comes from Harald Hammarström, who should be thanked for providing scans for version 2.4.

Apart from this, I am fine with the list, and we can merge it, but the last sentence in NOTE should be adjusted.

**Comment by LinguList** [comment]
I somehow forgot to follow this up. So if you are available, @tresoldi and @Kristina-Pianykh, could you have a quick look at the mappings? I select you, @tresoldi, as you will remember the list, which is at the origin of the Huber list.

**Comment by CarolinHu** [comment]
> 
> 
> @CarolinHu, the last sentence in the NOTE is no longer required: we have the PDF, did I share it? If not, can you remind me via email? I'll also prepare a PDF for the Concepticon. This comes from Harald Hammarström, who should be thanked for providing scans for version 2.4.
> 
> Apart from this, I am fine with the list, and we can merge it, but the last sentence in NOTE should be adjusted.

Great! I'll send a reminder. The list description was changed, and I also deleted the link to the page I got the data from.

File: concepticondata/conceptlists/Rowe-1954-232.tsv [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+Rowe-1954-232-82	82	olla, paila	cooking pot	3653	COOKING POT
+Rowe-1954-232-83	83	roza, chacra, conuco	cultivated clearing	212	FIELD
+Rowe-1954-232-84	84	caserío	village	930	VILLAGE
+Rowe-1954-232-85	85	camino, sendero	path, trail	2457	PATH OR ROAD
+Rowe-1954-232-86	86	red de pescar, chinchorro	fish net	388	FISHNET
+Rowe-1954-232-87	87	anzuelo	fishhook	1074	FISHHOOK
+Rowe-1954-232-88	88	hacha	axe	677	AXE
+Rowe-1954-232-89	89	cuchillo	knife	1352	KNIFE
+Rowe-1954-232-90	90	canoa, curiara	canoe	1970	CANOE
+Rowe-1954-232-91	91	remo	paddle	3137	OAR
```
Comment (Kristina-Pianykh) [inline concepticondata/conceptlists/Rowe-1954-232.tsv:92]: Hmm 'remo' in Huber is mapped to PADDLE but to OAR in Morse. Maybe @tresoldi can help here?

File: concepticondata/conceptlists/Rowe-1954-232.tsv [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+Rowe-1954-232-127	127	pájaro	bird	937	BIRD
+Rowe-1954-232-128	128	picaflor, colibrí, tucusito	humming bird	1589	HUMMINGBIRD
+Rowe-1954-232-129	129	guacamayo	macaw	1630	MACAW
+Rowe-1954-232-130	130	tucán, pinsha	toucan	308	TOUCAN
+Rowe-1954-232-131	131	loro	parrot	882	PARROT
+Rowe-1954-232-132	132	perico, pihuicho	parakeet	886	PARAKEET
+Rowe-1954-232-133	133	gallinazo, zamuro	buzzard	1865	BUZZARD
+Rowe-1954-232-134	134	pavo, guaco, paují	curassow	315	CURASSOW
+Rowe-1954-232-135	135	lechuza	owl	735	OWL
+Rowe-1954-232-136	136	paya de monte	guan	437	GUAN
```
Comment (Kristina-Pianykh) [inline concepticondata/conceptlists/Rowe-1954-232.tsv:137]: Should be 'pa**v**a de monte'.

File: concepticondata/conceptlists/Rowe-1954-232.tsv [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+Rowe-1954-232-144	144	zancudo	mosquito	1509	MOSQUITO
+Rowe-1954-232-145	145	comején, hormiga blanca	termite	883	TERMITE
+Rowe-1954-232-146	146	hormiga	ant	587	ANT
+Rowe-1954-232-147	147	araña	spider	843	SPIDER
+Rowe-1954-232-148	148	nigua, pique	chigger	438	CHIGGOE (JIGGER FLEA)
+Rowe-1954-232-149	149	monte	brush	1821	BRUSH
+Rowe-1954-232-150	150	pajonal, sabana	open grassland	260	SAVANNA
+Rowe-1954-232-151	151	cerro, loma	hill	614	HILL
+Rowe-1954-232-152	152	árbol	tree	906	TREE
+Rowe-1954-232-153	153	hoja	leaf	628	LEAF
```
Comment (Kristina-Pianykh) [inline concepticondata/conceptlists/Rowe-1954-232.tsv:154]: LEAF (LEAFLIKE OBJECT). Compare with Huber where we also can see the distinction between hoja and hoja de árbol). hoja de árbol should be mapped to LEAF then.

File: concepticondata/conceptlists/Rowe-1954-232.tsv [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+Rowe-1954-232-37	37	vulva	vulva	1837	VULVA
+Rowe-1954-232-38	38	hombre	man	1554	MAN
+Rowe-1954-232-39	39	varón, macho	man	2106	MALE PERSON
+Rowe-1954-232-40	40	mujer	woman	962	WOMAN
+Rowe-1954-232-41	41	gente	people	789	PEOPLE
+Rowe-1954-232-42	42	marido	husband	1200	HUSBAND
+Rowe-1954-232-43	43	esposa	wife	1199	WIFE
+Rowe-1954-232-44	44	papá	father	1217	FATHER
+Rowe-1954-232-45	45	mamá	mother	1216	MOTHER
+Rowe-1954-232-46	46	criatura	baby, infant	1289	BABY
```
Comment (Kristina-Pianykh) [inline concepticondata/conceptlists/Rowe-1954-232.tsv:47]: Nitpicking. In the source, the English equivalents are separate with` ; `.

File: concepticondata/conceptlists/Rowe-1954-232.tsv [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+Rowe-1954-232-47	47	viejo	old man	406	OLD MAN
+Rowe-1954-232-48	48	agua	water	948	WATER
+Rowe-1954-232-49	49	río	river	666	RIVER
+Rowe-1954-232-50	50	isla	island	622	ISLAND
+Rowe-1954-232-51	51	lago	lake	624	LAKE
+Rowe-1954-232-52	52	pantano	swamp	1145	SWAMP
+Rowe-1954-232-53	53	fuente; ojo de agua, manantial	spring of water	849	SPRING (OF WATER)
+Rowe-1954-232-54	54	caída de agua, catarata	waterfall	943	WATERFALL
+Rowe-1954-232-55	55	rabión, raudal	rapids	1670	RAPIDS
+Rowe-1954-232-56	56	candela, fuego	fogo
```
[the remark this answers is not in the document]
Comment (CarolinHu) [reply concepticondata/conceptlists/Rowe-1954-232.tsv]: That was probably a mistake by the site, in the original PDF the gloss is actually fire. I changed the gloss and added the mapping.

File: concepticondata/conceptlists/Rowe-1954-232.tsv (hunk 3 above)
Comment (CarolinHu) [reply concepticondata/conceptlists/Rowe-1954-232.tsv:154]: But is that really what the concept refers to? I thought hoja de árbol is a more specific kind of leaf (namely that of a tree compared to all other possible leaves), but hoja by itself still always means the botanical kind, not paper or similar objects. Some other lists map hoja to LEAF as well, so I'd like to wait for someone else to confirm or deny this mapping.

**Comment by CarolinHu** [comment]
Thank you for your review, @Kristina-Pianykh!
I changed some things according to your comments, now there's only hoja and remo to be decided. 
I will be on holiday for a week starting Monday, so I'll leave the update of the list descriptions of Morse, Mountain (and possibly Huber?) for a future PR.

**Comment by CarolinHu** [comment]
Since no merge conflicts arose during my absence, I added the Rowe reference to the list descriptions of the Mountain, Morse, and Huber here. @LinguList would you mind taking another look now?

**Comment by chrzyki** [comment]
Resolved the merge conflict. Looks fine now, thanks. :)

**Review by chrzyki — APPROVED** [review]

**Comment by CarolinHu** [comment]
Thank you! :)

**Pull request merged**